packages/exo/src/exo-tools.js

```diff
@@ -386,14 +372,11 @@ export const defendPrototype = (
     const {
       interfaceName,
       methodGuards: mg,
-      symbolMethodGuards,
```

Comment:
When getting rid of `symbolMethodGuards`, need to instead accommodate `@@asyncIterator` as a reserved method guard name. That's weird.

Need deprecation dance

Reply:
Reverted in this PR